packages/preview/yats/0.1.0/README.md

```diff
@@ -0,0 +1,88 @@
+# Yats
+
+serialize the Typst objects into bytes and deserialize the bytes into Typst objects
+
+## Reason
+I want to interactive between the wasm and typst. But I found that the input arguments and output argument are all bytes. It is not convenient for me to use WASM. So I designed the serialization protocol and implemented this serialization module for reference.
+
+Although there have been some serialization APIs like cbor, yaml, json and others, this is a good learning material and a good example to show the abilities of Typst.
+
+## Example
+
+Have a look at the example [here](./example.typ).
+
+## Usage
+
+Simply import 2 functions : `serialize`, `deserialize`.
+
+And enjoy it
+
+```typ
+#import "yats.typ": serialize,deserialize
```

Comment:
```suggestion
#import "@preview/yats:0.1.0": serialize, deserialize
```
it's better if the README (and ideally also the `example.typ`) shows an import from the package repository

Reply:
I updated both files. You can check.